apps/www/server/index.js

```diff
@@ -20,6 +20,10 @@ const { CURTAIN_MESSAGE } = process.env
 
 const app = next({
   dev: DEV,
+  port: PORT,
+  hostname: process.env.PUBLIC_BASE_URL
+    ? new URL(process.env.PUBLIC_BASE_URL).hostname
+    : 'localhost',
```

Comment:
maybe it would be better to fall back to `undefined`? Or even refuse to start the server without `PUBLIC_BASE_URL` since it's necessary?

Reply:
Good catch! 👍🏻